src/index.html

```diff
@@ -3,11 +3,402 @@
+            class="by-category__sunglasses-black"
+          >
+
+          <p class="by-category__name">
+            Audio sunglasses
+          </p>
+        </section>
+
+        <div class="by-category__button-wrapper">
+          <a href="#" class="by-category__button button">
+            View all
+          </a>
+        </div>
+      </article>
+
+      <article class="testimonials">
+        <section class="testimonials__wrapper">
+          <h2
+            class="title testimonials__title"
+            id="testimonials"
+          >
+            Why buy direct from Bose
+          </h2>
+
+          <div class="testimonials__text">
+            <p class="testimonials__line">
+              Free 2-day shipping and returns
+            </p>
+
+            <p class="testimonials__line">
+              90-day risk-free trial
+            </p>
+
+            <p class="testimonials__line">
+              World class customer service
+            </p>
+
+            <p class="testimonials__line">
+              My Bose account management
+            </p>
+
+            <p class="testimonials__parag">
+              A great product is more than what’s in the box. It’s a promise of premium performance, world-class support, and everything you expect from a trusted brand. It’s just one of many reasons why you’ll shop with confidence on Bose.com.
+            </p>
+          </div>
+        </section>
+      </article>
+
+      <article class="contact">
+        <h2
+          class="contact__title title"
+          id="contact"
+        >
+          Contact us
+        </h2>
+
+        <section class="contact__wrapper">
+          <div class="contact__form-wrapper">
+            <form
+              action="#"
+              method="get"
+              class="contact__form"
+              id="contact-form"
+            >
+              <label for="name"></label>
+              <input
+                type="text"
+                name="name"
+                id="name"
+                placeholder="Name"
+                class="contact__input contact__input--first"
+                minlength="5"
+                required
+              >
+
+              <label for="email"></label>
+              <input
+                type="email"
+                name="email"
+                id="email"
+                placeholder="Email@email.com"
+                class="contact__input"
+                required
+              >
+
+              <label for="message"></label>
+              <textarea
+                class="contact__input contact__input--text-area"
+                name="message"
+                id="message"
+                placeholder="Message"
+                minlength="5"
+                required
+              ></textarea>
+            </form>
+
+            <button
+              type="submit"
+              class="button contact__button"
+              form="contact-form"
+            >
+              Send
+            </button>
```

Comment:
oh and here - why do you drag the button out of the form?
it belongs there - let it stick with it's pack 🙂